### ISSUE TITLE
how many a100 it cost in training and if i want to train on v100 what is the number needed ?

### ISSUE THREAD

**Issue by Yang-bug-star** [OpenMOSS/AnyGPT#4]

**Comment by JunZhan2000** [comment]
> Could you please specify whether you used the 40GB or 80GB version of the A100s to train your model?

80G

**Comment by gsq7474741** [comment]
> In the pre-training phase, we used 96 A100 GPUs, with 4500 tokens per gpu, set the gradient accumulation steps to 8, and trained for 81k steps. The training took about one week. The computational resources required for the instruction tuning phase are much less.

btw, how many A100 were used in the tuning phase？
